### PR TITLE
Update for CDAP Release 3.5.0

### PR DESCRIPTION
**Pull request by awholegunch**
Updates to READMEs due to changes in CDAP 3.5.0.

### CODE REVIEW

File: DBTableToCDAPTable/README.rst [hunk 1]
```diff
@@ -41,7 +51,7 @@ Configurations for the Database Table Source
 
 #. ``splitBy``: The is the field used to generate splits. Not required if numSplits is set to one.
 
-#. ``numSplits``: The number of splits to generate.
+#. ``numSplits``: A string of the number of splits to generate (optional).
```
Comment (albertshau) [inline DBTableToCDAPTable/README.rst]: everything is a string. Do we need to explicitly say this?

File: DBTableToCDAPTable/config.json [hunk 2]
```diff
@@ -8,19 +8,21 @@
...
-            "importQuery": "select col1,col2,col3,col4 from my_table where $CONDITIONS",
-            "splitBy": "col1",
-            "boundingQuery": "select min(col1), max(col1) from my_table",
+            "referenceName": "dbSource",
             "connectionString": "jdbc:postgresql://localhost:5432/testdb",
             "user": "postgres",
             "password": "",
+            "importQuery": "select col1,col2,col3,col4 from my_table where $CONDITIONS",
+            "splitBy": "col1",
+            "numSplits": 1,
```
Comment (albertshau) [inline DBTableToCDAPTable/config.json]: 1 -> "1"

**Comment by albertshau** [comment]
minor comments. otherwise lgtm for readmes. They might require other updates but we can do those in a separate PR.

File: DBTableToCDAPTable/README.rst (hunk 1 above)
Comment (awholegunch) [reply DBTableToCDAPTable/README.rst]: I don't think we ever explicitly say that. JSON itself does not require strings, but do we in _all_ our JSON?

If so, somewhere we should say that.

File: DBTableToCDAPTable/config.json (hunk 2 above)
Comment (awholegunch) [reply DBTableToCDAPTable/config.json]: Fixed.

File: DBTableToCDAPTable/README.rst (hunk 1 above)
Comment (awholegunch) [reply DBTableToCDAPTable/README.rst]: Added as https://issues.cask.co/browse/CDAP-7180. Will remove from here to complete this PR.

File: RealtimeTwitterToHBase/README.rst [hunk 3]
```diff
@@ -1,24 +1,32 @@
 ====================================================
-Real-time Twitter To HBase Application Configuration
+Real-time Twitter to HBase Application Configuration
 ====================================================
 
-Introduction
-------------
+Let's say we want to fetch Tweets from Twitter in real time and persist them in an HBase
+Table. The ``cdap-etl-realtime`` system artifact can be used to create an Application that
```
Comment (wolf31o2) [inline RealtimeTwitterToHBase/README.rst]: Should this be `cdap-data-streams`?

**Comment by wolf31o2** [comment]
One comment, otherwise 👍 LGTM

**Comment by awholegunch** [comment]
This PR is "on-hold" as there are a number of errors still in the READMEs, plus everything should be tested.

File: RealtimeTwitterToHBase/config.json [hunk 4]
```diff
@@ -1,35 +1,69 @@
...
+            "version": "1.4.1",
+            "scope": "SYSTEM"
+          },
           "properties": {
-            "name": "tweetTable",
-              "schema.row.field": "id",
-              "schema": "{\"type\":\"record\",\"name\":\"tweet\",\"fields\":[{\"name\":\"id\",\"type\":\"long\"},{\"name\":\"message\",\"type\":\"string\"},{\"name\":\"lang\",\"type\":[\"string\",\"null\"]},{\"name\":\"time\",\"type\":[\"long\",\"null\"]},{\"name\":\"favCount\",\"type\":\"int\"},{\"name\":\"rtCount\",\"type\":\"int\"},{\"name\":\"source\",\"type\":[\"string\",\"null\"]},{\"name\":\"geoLat\",\"type\":[\"double\",\"null\"]},{\"name\":\"geoLong\",\"type\":[\"double\",\"null\"]},{\"name\":\"isRetweet\",\"type\":\"boolean\"}]}"
+            "referenceName": "twitterSource",
+            "ConsumerKey": "GetConsumerKeyFromTwitter",
+            "ConsumerSecret": "GetConsumerSecretFromTwitter"
```
Comment (albertshau) [inline RealtimeTwitterToHBase/config.json]: missing a comma here. Guess that's why the ':' in the next line is highlighted in red?

**Review by albertshau — CHANGES_REQUESTED** [review]
one small comment, otherwise lgtm

File: RealtimeTwitterToHBase/config.json (hunk 4 above)
Comment (awholegunch) [reply RealtimeTwitterToHBase/config.json]: Probably.

Comment (awholegunch) [reply RealtimeTwitterToHBase/config.json]: That fixed it.

**Pull request merged**